utilities/unittests/test_hco.py

```
@@ -583,52 +581,57 @@ def test_resource_editor_init(self, mock_namespace_class, mock_get_client):
         )
 
+        assert editor.admin_client == mock_client
         assert editor.hco_namespace == mock_namespace
         assert editor._consecutive_checks_count == 5
         assert editor.list_resource_reconcile == []
 
     @patch("utilities.hco.wait_for_hco_conditions")
-    @patch("utilities.hco.get_client")
     @patch("utilities.hco.Namespace")
-    def test_resource_editor_update_without_reconcile(self, mock_namespace_class, mock_get_client, mock_wait_hco):
+    def test_resource_editor_update_without_reconcile(self, mock_namespace_class, mock_wait_hco):
```

Ruff (0.15.21)

[warning] Unused method argument: `mock_namespace_class`

(ARG002)